Source/missiles.cpp

```diff
@@ -4931,8 +4857,7 @@ void ProcessMissiles()
 		}
 	}
 
-	i = 0;
-	while (i < nummissiles) {
+	for (int i = 0; i < nummissiles;) {
 		if (missile[missileactive[i]]._miDelFlag) {
 			DeleteMissile(missileactive[i], i);
 			i = 0;

```

Comment:
```suggestion
```
This assignment is unnecessary, the DeleteMissile function only modifies `missileactive[i]`